src/simple-cache-client.ts

```diff
@@ -54,14 +54,12 @@ export class SimpleCacheClient {
   private readonly dataClients: Array<CacheClient>;
   private nextDataClientIndex: number;
   private readonly controlClient: ControlClient;
-  private readonly logger: Logger;
 
   /**
    * Creates an instance of SimpleCacheClient.
    */
   constructor(props: SimpleCacheClientProps) {
     initializeMomentoLogging(props.configuration.getLoggerOptions());
-    this.logger = getLogger(this);
```

Comment:
Instead of removing this can you just add a debug log message to the constructor or something?  We will be coming back in here and adding logging at some point and I want to make sure it's really easy for devs to figure out how to do it.

Reply:
I went to do that, and there's very little to log in SimpleCacheClient. All the work is in CacheClient and ControlClient. I think this is at the wrong level.